docs/source/tutorials/raw_key.rst

```diff
@@ -20,7 +20,7 @@ Synopsis
 
 **key_info_string** is of the form::
 
-    label=<label>:key_id=<key_id>:key=<key>
+    label=<label>:key_id=<key_id>:key=<key>[:iv=<initialization_vector>]
```

Comment:
Can you update the one of the examples below to include the new field, e.g. on "Example with FairPlay using 'cbcs' protection scheme::"?